File: minegame-server/src/main/java/top/nipuru/minegame/server/game/setting/SettingData.java
package top.nipuru.minegame.server.game.setting;

import top.nipuru.minegame.server.game.player.Data;
import top.nipuru.minegame.server.game.player.Table;
import top.nipuru.minegame.server.game.player.Unique;

@Table(name = "tb_setting")
public class SettingData implements Data {

    /** 设置 id */
    @Unique int id;

    /** 选项值 */
    int option;
}

File: minegame-game/src/main/kotlin/top/nipuru/minegame/game/gameplay/setting/SettingData.kt
package top.nipuru.minegame.game.gameplay.setting

import top.nipuru.minegame.game.gameplay.player.Data
import top.nipuru.minegame.game.gameplay.player.Table
import top.nipuru.minegame.game.gameplay.player.Unique

@Table(name = "tb_setting")
class SettingData : Data {
    /** 设置 id  */
    @JvmField
    @Unique
    var id: Int = 0

    /** 选项值  */
    var option: Int = 0
}
